<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1E1E1E;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnSAT">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>290</y>
      <width>1022</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #C9DFEE;
background-color:  #0A84FF;
border-radius: 12px;
text-align: left;
padding-left: 170 px;
</string>
    </property>
    <property name="text">
     <string>Solder Ability Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTM">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>470</y>
      <width>1022</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #C9DFEE;
background-color:  #0A84FF;
border-radius: 12px;
text-align: left;
padding-left: 170 px;
</string>
    </property>
    <property name="text">
     <string>Thickness measurement</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCA">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>1022</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #C9DFEE;
background-color:  #0A84FF;
border-radius: 12px;
text-align: left;
padding-left: 170 px;
</string>
    </property>
    <property name="text">
     <string>Chemical Ananlysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>1021</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>58</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:	None;

color: #F2F2F2;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Please Select&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="iconCA">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C9DFEE;
border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/chemical_Analysis.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconSAT">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C9DFEE;
border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/solder_ability_test.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconTM">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C9DFEE;
border-radius:10px
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/tape-measure.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGL">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>650</y>
      <width>1022</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #C9DFEE;
background-color:  #0A84FF;
border-radius: 12px;
text-align: left;
padding-left: 170 px;
</string>
    </property>
    <property name="text">
     <string>Group Lead</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconGL">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>660</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C9DFEE;
border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/Group_lead.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMGR">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>830</y>
      <width>1022</width>
      <height>160</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #C9DFEE;
background-color:  #0A84FF;
border-radius: 12px;
text-align: left;
padding-left: 170 px;
</string>
    </property>
    <property name="text">
     <string>Manager</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="iconMGR">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>840</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #C9DFEE;
border-radius:10px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Image/Manager.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closebtnSelect">
    <property name="geometry">
     <rect>
      <x>1819</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Image/cross.png</normaloff>Image/cross.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>65</width>
      <height>65</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
